hack/run-ci-e2e-test.sh

```diff
@@ -46,15 +42,43 @@ NODE_COUNT=${NODE_COUNT:-2}
 SKIP_NODE_DELETION=${SKIP_NODE_DELETION:-"-skip-node-deletion=false"}
 KEY_PAIR_NAME=${KEY_PAIR_NAME:-"libra"}
 
+# OPERATOR_IMAGE defines where the WMCO image to test with is located. If $OPERATOR_IMAGE is already set, use its value.
+# In OpenShift CI $IMAGE_FORMAT will be set to a value such as registry.svc.ci.openshift.org/ci-op-<input-hash>/stable:${component}
+# We need to replace ${component} with the name of the WMCO image.
+# The stable namespace is used as it will use the last promoted version when the test is run in an repository other than this one.
+# When running in this repository the image built in the pipeline will be used instead.
+# https://ci-operator-configresolver-ui-ci.apps.ci.l2s4.p1.openshiftapps.com/help/ci-operator#release
+OPERATOR_IMAGE=${OPERATOR_IMAGE:-${IMAGE_FORMAT//\/stable:\$\{component\}//stable:windows-machine-config-operator-test}}
+
+MANIFEST_LOC="/tmp/ci-wmco-manifest"
+cp -r deploy/olm-catalog/windows-machine-config-operator/manifests $MANIFEST_LOC
```

Comment:
How is this working? I don't see a `manifests` directory in the repo.

Reply:
That will be added in #54 and is why this is blocked by it